libs/components/lists/src/lib/modules/sort/sort-item.component.ts

```diff
@@ -30,63 +30,66 @@ export class SkySortItemComponent implements OnInit, OnChanges, OnDestroy {
    * Indicates whether the sorting option is active.
    */
   @Input()
-  public active: boolean;
+  public active: boolean | undefined;
 
   /**
    * Fires when a sort item is selected.
    */
   @Output()
-  public itemSelect: EventEmitter<any> = new EventEmitter();
+  public itemSelect: EventEmitter<any> = new EventEmitter(); // TODO: EventEmitter is of type <void>
 
   public isSelected: BehaviorSubject<boolean> = new BehaviorSubject<boolean>(
     false
   );
 
-  private subscription: Subscription;
+  #subscription!: Subscription;
 
-  private sortItemId: string;
+  #sortItemId!: string;
 
-  constructor(
-    private sortService: SkySortService,
-    private detector: ChangeDetectorRef
-  ) {}
+  #sortService: SkySortService;
+  #detector: ChangeDetectorRef;
 
-  public ngOnInit() {
+  constructor(sortService: SkySortService, detector: ChangeDetectorRef) {
+    this.#sortService = sortService;
+    this.#detector = detector;
+  }
+
+  ngOnInit(): void {
```

Comment:
Needs `public`.

Reply:
fixed in: [respond to comment](https://github.com/blackbaud/skyux/pull/686/commits/79dceb37a5310453c7a1ceee9ce8171431f6d3e2)